tests/Microsoft.VisualStudio.ProjectSystem.Managed.UnitTests/ProjectSystem/UpToDate/BuildUpToDateCheckTests.cs

```diff
@@ -499,7 +503,7 @@ public async Task InitialItemDataDoesNotUpdateLastItemsChangedAtUtc()
                 [Compile.SchemaName] = SimpleItems("ItemPath1", "ItemPath2")
             };
 
-            await _buildUpToDateCheck.LoadAsync();
+            await _buildUpToDateCheck.TestAccess.InitializedTask;
```

Comment:
> await _buildUpToDateCheck.TestAccess.InitializedTask; [](start = 11, length = 54)

should this be a method to simplify test updates if the initialization logic changes?

Reply:
Sure, I've changed it to a method. These `TestAccess` members are all internal so we can change them without worrying about compat.